content/src/content/jcr_root/apps/core/wcm/components/commons/editor/clientlibs/parsysplaceholder/js.txt

```diff
@@ -0,0 +1,20 @@
+###############################################################################
+# Copyright 2020 Adobe
```

Comment:
as we get that merged in 2021 should we use 2021 in all copyrights?

Reply:
fixed with https://github.com/adobe/aem-core-wcm-components/pull/1351/commits/d3e123e5d2b219d083b276ea28e55944f7e2e01e